documentation/src/main/asciidoc/topics/proc_cli_connecting.adoc

```diff
@@ -1,25 +1,43 @@
+[id='cli_start-{context}']
 = Connecting to {brandname} Servers
+Establish CLI connections to {brandname}.
 
-Do one of the following:
+.Prerequisites
 
-* Run the `connect` command to connect to a {brandname} server on the default port of `11222`:
+Add user credentials and have at least one running {brandname} server instance.
+
+.Procedure
+
+. Open a terminal in `{server_home}`.
+. Start the CLI.
++
+* *Linux:*
++
+[source,options="nowrap",subs=attributes+]
+----
+include::cmd_examples/cli_nix.adoc[]
+----
++
+* *Microsoft Windows:*
++
+[source,options="nowrap",subs=attributes+]
+----
+include::cmd_examples/cli_win.adoc[]
+----
++
+. Enter your username and password when prompted.
```

Comment:
the username and password are prompted after the `connect` command.
```
$ ./server/runtime/target/infinispan-server-11.0.1-SNAPSHOT/bin/cli.sh 
[disconnected]> connect
Username: dev
Password: ***
```

Reply:
I would probably merge it in the next bullet:
"Run the [command]`connect` command and enter your username and password when prompted."

Reply:
offtopic: what does `[command]` do?

Reply:
another good comment. `[command]` adds syntax highlighting downstream. I don't have an example to hand but it's in the markup conventions at: https://redhat-documentation.github.io/asciidoc-markup-conventions/